database_schemas/cassandra/benchflow.cql

```diff
@@ -135,7 +134,7 @@ CREATE TABLE container_properties (
   name text,
   network text,
   restart_policy text,
-  PRIMARY KEY ((trial_id, experiment_id), container_properties_id, host_id)
+  PRIMARY KEY ((trial_id, experiment_id), container_id)
 );
```

Comment:
Rolling back https://github.com/benchflow/benchflow/pull/25/files/c1b43260cec94f30b487372e66bd871f413bc904..39512418abd06d7696fc867e2305abfe149f05b3, means adding back `container_properties_id uuid` and using `container_properties_id, host_id` as part of the group key.